src/components/PatchActionButtons/SetPatchPriority.tsx

```diff
@@ -59,7 +59,7 @@ export const SetPatchPriority: React.FC<SetPriorityProps> = ({
           <StyledBody>Set new priority:</StyledBody>
           <InputNumber
             size="small"
-            min={0}
+            min={-1}
```

Comment:
The ability to disable a task was added in https://github.com/evergreen-ci/spruce/commit/b2552477b194f328e41359609be5b8243a3360f0. I don't think it makes sense to have 2 ways to do it, and I intentionally didn't go this route because the dropdown option is clearer